java/ql/lib/semmle/code/java/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -150,8 +150,7 @@ predicate readStep(Node node1, Content f, Node node2) {
  * in `x.f = newValue`.
  */
 predicate clearsContent(Node n, Content c) {
-  c instanceof FieldContent and
-  n = any(PostUpdateNode pun | storeStep(_, c, pun)).getPreUpdateNode()
+  exists(FieldAccess fa | instanceFieldAssign(_, fa) and n = getFieldQualifier(fa))
```

Comment:
Isn't a problem that `c` is now unbound here?

Reply:
Good spot!

Reply:
Good catch. That's a bug.